Modix.Services/Messages/Discord/ChatMessageReceived.cs

```diff
@@ -0,0 +1,15 @@
+using Discord;
+using MediatR;
+
+namespace Modix.Services.Messages.Discord
+{
+    public class ChatMessageReceived : INotification
+    {
+        public ChatMessageReceived(IMessage message)
+        {
+            Message = message;
+        }
+
+        public IMessage Message { get; }
```

Comment:
I'd just assume make these publicly settable, like all other model objects in MODiX. Do we really not trust our devs? Also, I think `{ }` initializer syntax is cleaner.